### PR TITLE
Added staging values

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1218

### Change description ###
• This PR overrides queue connection string secrets with staging infra ones.
• Other secrets remain same but needs to be present while overriding.
• This PR is dependent on hmcts/bulk-scan-shared-infrastructure#169 to create secrets in key vault.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/bulk-scan-orchestrator/values.yaml [hunk 1]
```diff
@@ -38,8 +38,11 @@ java:
         - idam-users-sscs-username
         - idam-users-sscs-password
         - envelopes-queue-listen-connection-string
+        - envelopes-staging-queue-listen-connection-string
```
Comment (mdayican) [inline charts/bulk-scan-orchestrator/values.yaml:41]: you do not need any staging in values.yaml
just add them to values.aat.yaml

File: charts/bulk-scan-orchestrator/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: bulk-scan-orchestrator
 apiVersion: v1
 home: https://github.com/hmcts/bulk-scan-orchestrator
-version: 0.2.9
+version: 0.2.10
```
Comment (mdayican) [inline charts/bulk-scan-orchestrator/Chart.yaml:4]: you will not need to bump the chart. as you will not add anything to values.yaml

File: charts/bulk-scan-orchestrator/values.aat.template.yaml [hunk 3]
```diff
@@ -3,3 +3,10 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   devApplicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
+  keyVaults:
+    bulk-scan:
+      resourceGroup: bulk-scan
+      secrets:
```
Comment (nitinprabhu) [inline charts/bulk-scan-orchestrator/values.aat.template.yaml:9]: Secret section should include all keys please refer other repos to see how it needs to be configured.

**Comment by nitinprabhu** [comment]
> ### JIRA link (if applicable)
> https://tools.hmcts.net/jira/browse/BPS-1218
> 
> ### Change description
> • This PR overrides queue connection string secrets with staging infra ones.
> • Other secrets remain same but needs to be present while overriding.
> • This PR is dependent on [hmcts/bulk-scan-shared-infrastructure#168](https://github.com/hmcts/bulk-scan-shared-infrastructure/pull/168) to create secrets in key vault.
> 
> **Does this PR introduce a breaking change?** (check one with "x")
> 
> ```
> [ ] Yes
> [x] No
> ```

PR mentioned in the description is closed ?

**Comment by apereverzin** [comment]
Fixed description

**Comment by doncem** [comment]
I thought it was ready but linked PR for infra changes is not merged

**Pull request merged**